hw-event-proxy/go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	github.com/json-iterator/go v1.1.11
 	github.com/redhat-cne/sdk-go v0.1.1-0.20210723134838-a808941830c9
```

Comment:
you  might want to run  go get github.com/redhat-cne/sdk-go@main

Reply:
updated